styles.css

```diff
@@ -0,0 +1,179 @@
+@import url('https://fonts.googleapis.com/css?family=Monoton|Noto+Serif+KR|Black+Ops+One|Press+Start+2P|Orbitron|VT323');
```

Comment:
I did not know where is a better place to leave this comment: maybe more commit.